com.unity.render-pipelines.universal/Editor/Camera/UniversalRenderPipelineCameraEditor.cs

```diff
@@ -583,134 +457,46 @@ void DrawStackSettings()
-                }
-
-                if (m_OutputWarningCameras.Any())
-                {
-                    var message = new StringBuilder();
-                    message.Append("The output properties of this Camera do not match the output properties of the following Cameras: ");
-                    foreach (var camera in m_OutputWarningCameras)
-                    {
-                        message.Append(camera.name);
-                        if (camera != m_OutputWarningCameras.Last())
-                            message.Append(", ");
-                        else
-                            message.Append(".");
-                    }
-
-                    CoreEditorUtils.DrawFixMeBox(message.ToString(), MessageType.Warning, () => UpdateStackCamerasOutput());
-                }
-                EditorGUI.indentLevel++;
+            EditorGUILayout.Space();
 
-                EditorGUILayout.Space();
-                EditorGUILayout.Space();
-            }
-            EditorGUILayout.EndFoldoutHeaderGroup();
-        }
+            m_LayerList.DoLayoutList();
+            m_SerializedCamera.Apply();
 
-        void DrawEnvironmentSettings(CameraRenderType camType)
-        {
-            m_EnvironmentSettingsFoldout.value = EditorGUILayout.BeginFoldoutHeaderGroup(m_EnvironmentSettingsFoldout.value, Styles.environmentSettingsText);
-            if (m_EnvironmentSettingsFoldout.value)
+            EditorGUI.indentLevel--;
+            if (m_TypeErrorCameras.Any())
             {
-                if (camType == CameraRenderType.Base)
+                var message = new StringBuilder();
+                message.Append("The type of the following Cameras must be Overlay render type: ");
+                foreach (var camera in m_TypeErrorCameras)
                 {
-                    DrawClearFlags();
-                    if (!settings.clearFlags.hasMultipleDifferentValues)
-                    {
-                        if (GetBackgroundType((CameraClearFlags)settings.clearFlags.intValue) == BackgroundType.SolidColor)
-                        {
-                            using (var group = new EditorGUILayout.FadeGroupScope(m_ShowBGColorAnim.faded))
-                            {
-                                if (group.visible)
-                                {
-                                    settings.DrawBackgroundColor();
-                                }
-                            }
-                        }
-                    }
+                    message.Append(camera.name);
+                    if (camera != m_TypeErrorCameras.Last())
+                        message.Append(", ");
+                    else
+                        message.Append(".");
                 }
-                DrawVolumes();
-                EditorGUILayout.Space();
-                EditorGUILayout.Space();
+
+                CoreEditorUtils.DrawFixMeBox(message.ToString(), MessageType.Error, () => UpdateStackCemerasToOverlay());
```

Comment:
For a future PR but i noticed on your screenshot that the text was hidden behind the button.